.github/workflows/integrate-rc-to-develop.yaml

```diff
@@ -0,0 +1,28 @@
+name: RC to Develop Integration
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 7 * * *" # Every day at 07:30 AM UTC - equivalent to 07:30 PM NZST (or 08:30 PM NZDT)
+
+jobs:
+  integrate-rc-to-develop:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.NIGHTLY_BUILD_GITHUB_PAT }}
```

Comment:
Need  a PAT that have access for creating PRs and editing branches. We can create a new one, but from memory this PAT has the right access already so thought to just reuse

Reply:
Yeah I think this is fine better not have too many PATs!